test/integration/programs/list/programs-all.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'underscore';
+import moment from 'moment';
+
+context('program all list', function() {
+  specify('display programs list', function() {
+    cy
+      .server()
+      .routePrograms(fx => {
+        fx.data = _.sample(fx.data, 3);
+
+        fx.data[0] = {
+          id: '1',
+          attributes: {
+            name: 'First in List',
+            published: true,
+            updated_at: moment().format(),
+          },
+        };
+
+        fx.data[1].attributes = {
+          name: 'Last in List',
+          published: true,
+          updated_at: moment().subtract(2, 'days').format(),
+        };
+
+        fx.data[2].attributes = {
+          name: 'Secomd in List, Not Published',
+          published: false,
+          updated_at: moment().subtract(1, 'day').format(),
+        };
+
+        return fx;
+      })
+      .visit('/programs');
+
+    cy
+      .get('.table-list__item')
+      .first()
+      .should('contain', 'First in List')
+      .should('contain', 'On');
+
```

Comment:
you _could_ just `.next()` here and I think it's be ok.. the context doesn't change.